lib/resource-mapper.js

```diff
@@ -24,6 +25,7 @@ class ResourceMapper {
     this._defaultContentType = defaultContentType
     this._indexFilename = indexFilename
     this._types = { ...types, ...overrideTypes }
+    this._isControlFile = new RegExp(`(?:${fileSuffixes.map(fs => fs.replace('.', '\\.')).join('|')})$`)
```

Comment:
`fs.replace(/\./g, '\\.')` strictly, but good enough 😉

Reply:
Regex is so difficult =P